main.py

```diff
@@ -1,15 +1,36 @@
 import random
-userpick = input("What number from 0 to 100 I am thinking about? ")
-picked = int(random.randrange(0 , 100))
-print(f"userpick: {userpick}")
-print(f"picked: {picked}")
-number_of_tries = 1
-while userpick != picked:
-    picked = int(random.randrange(0 , 100))
-    userpick = input("Not this one. Try another one ")
-    print(f"userpick: {userpick}")
-    print(f"picked: {picked}")
-    number_of_tries += 1
-else:
-    name = str(input("You made it! Let`s save your score. Give me your nick: "))
+
+def comparing_selection(choosed: int, random: int):
+    if choosed > random:
+        print ("Try smaller one")
+        return False
+    elif choosed < random:
+        print("I am thinking about bigger one")
+        return False
+    else:
+        return True
+    
+def write_to_file(name: str, number_of_tries: int):
```

Comment:
Add functions description:
```
"""
Archiving game statistics by writing them to the file.

:param name: Gamer's nickname
:param number_of_tries: Number of tries to guess a number
"""
```
If any function has `return` in docstring add `:return: Describe what is returned`